lua/obsidian/search.lua

```diff
@@ -36,6 +36,7 @@ M.Patterns = {
 
   -- Miscellaneous
   Highlight = "==[^=]+==", -- ==text==
+  Callout = "%[!.*%]",
```

Comment:
Shouldn't callouts always start with ">"?

Reply:
Ah that's right